content/zh/docs/concepts/observability-primer.md

```diff
@@ -35,11 +35,11 @@ cSpell:ignore: webshop
 
 分布式链路让你能够观察请求如何在复杂的分布式系统中传播。它提高了应用程序或系统健康状况的可见性，并让你能够调试那些难以在本地重现的行为。对于分布式系统来说，分布式链路是必不可少的，因为这些系统通常存在不确定性问题，或者过于复杂而无法在本地重现。
 
-要理解分布式链路，你需要了解其各个组成部分的角色：日志、span（跨度）和 trace（链路）。
+要理解分布式链路，你需要了解其各个组成部分的角色：日志（logs）、跨度（spans）和 链路（traces）。
 
 ### 日志
```

Comment:
Add a heading ID here. For details, see https://opentelemetry.io/docs/contributing/localization/#headings.